### PR TITLE
fix(console): optimize loading screens in app guides

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Adjust loading screens when loading app guide content.

## Testing

Locally tested, no details page skeleton page was observed.

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] docs~~

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/ApplicationDetails/index.tsx [hunk 1]
```diff
@@ -145,10 +145,23 @@ function ApplicationDetails() {
...
+        }}
+      />
+    );
+  }
+
   return (
     <DetailsPage
       backLink="/applications"
       backLinkTitle="application_details.back_to_applications"
+      // Hide app details loading skeleton if it's loading guide
```
Comment (gao-sun) [inline packages/console/src/pages/ApplicationDetails/index.tsx:164]: this comment stands out but seems not matching the code below

Comment (charIeszhao) [reply packages/console/src/pages/ApplicationDetails/index.tsx:164]: I remember I removed it, don't know why it came back... Will remove again

File: packages/console/src/pages/ApplicationDetails/index.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ const mapToUriOriginFormatArrays = (value?: string[]) =>
 function ApplicationDetails() {
   const { id, guideId, tab } = useParams();
   const { navigate, match } = useTenantPathname();
-  const isGuideView = id && guideId && match(`/applications/${id}/guide/${guideId}`);
+  const isGuideView = !!id && !!guideId && match(`/applications/${id}/guide/${guideId}`);
```
Comment (wangsijie) [inline packages/console/src/pages/ApplicationDetails/index.tsx:55]: nit: use `Boolean()`?

File: packages/console/src/pages/Applications/components/Guide/index.tsx [hunk 3]
```diff
@@ -64,27 +64,30 @@ function Guide({ className, guideId, app, onClose }: Props) {
...
-          endpoint: tenantEndpoint?.toString() ?? '',
-          alternativeEndpoint: conditional(isCustomDomainActive && tenantEndpoint?.toString()),
-          redirectUris: app.oidcClientMetadata.redirectUris,
-          postLogoutRedirectUris: app.oidcClientMetadata.postLogoutRedirectUris,
-          sampleUrls: {
-            origin: 'http://localhost:3001/',
-            callback: 'http://localhost:3001/callback',
-          },
-        }
+        !!guide &&
```
Comment (wangsijie) [inline packages/console/src/pages/Applications/components/Guide/index.tsx:67]: ditto

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/console/src/pages/ApplicationDetails/index.tsx (hunk 2 above)
Comment (charIeszhao) [reply packages/console/src/pages/ApplicationDetails/index.tsx:55]: No, using `Boolean()` cannot rule out the possibility of `undefined` in the right hand side expression.

image

File: packages/console/src/pages/Applications/components/Guide/index.tsx (hunk 3 above)
Comment (charIeszhao) [reply packages/console/src/pages/Applications/components/Guide/index.tsx:70]: Same

**Pull request merged**